### PR TITLE
merge and add main activity ui

### PR DESCRIPTION
**Pull request by Csoyee**
I merged and add some UI component in main activity

### CODE REVIEW

File: Roys/app/src/main/java/com/dwg_karrier/roys/Authentication.java [hunk 1]
```diff
@@ -65,7 +65,7 @@ static String convertStreamToString(InputStream is) {
   }
 
   void authenticationAndBringPages() {
-    Dialog authDialog = new Dialog(mainContext);
+    final Dialog authDialog = new Dialog(mainContext);
```
Comment (xhae) [inline Roys/app/src/main/java/com/dwg_karrier/roys/Authentication.java]: Is `authDialog` need to be final?

File: Roys/app/src/main/java/com/dwg_karrier/roys/ContentSwipe.java [hunk 2]
```diff
@@ -160,6 +160,7 @@ public SectionsPagerAdapter(FragmentManager fm) {
 
     @Override
     public Fragment getItem(int position) {
+
```
Comment (xhae) [inline Roys/app/src/main/java/com/dwg_karrier/roys/ContentSwipe.java]: nit: erase

File: Roys/app/src/main/java/com/dwg_karrier/roys/MainActivity.java [hunk 3]
```diff
@@ -5,41 +5,42 @@
...
-    btnFeedlyAccount.setOnClickListener(new Button.OnClickListener() {
-      public void onClick(View v) {
-        Authentication authentication = new Authentication(mainActivity);
-        authentication.authenticationAndBringPages();
-      }
-    });
-
-    GridLayout minuteLayout = (GridLayout) findViewById(R.id.mainLayout);
-    setImage();
+    final GridLayout minuteLayout = (GridLayout)findViewById(R.id.maingridLayout);
```
Comment (xhae) [inline Roys/app/src/main/java/com/dwg_karrier/roys/MainActivity.java]: Same final issue here. Final is usually used to identify constant values.

Comment (Csoyee) [inline Roys/app/src/main/java/com/dwg_karrier/roys/MainActivity.java]: No problem! Maybe it was added automatically during testing.
I'll erase it.

File: Roys/app/src/main/java/com/dwg_karrier/roys/Authentication.java (hunk 1 above)
Comment (Csoyee) [inline Roys/app/src/main/java/com/dwg_karrier/roys/Authentication.java]: In onPageFinished function error occurs that authDialog need to be final as that varaiable is accessed from within inner class. 

I'll check it!!

**Comment by Csoyee** [comment]
I applied comment!

File: Roys/app/src/main/java/com/dwg_karrier/roys/MainActivity.java [hunk 4]
```diff
@@ -5,41 +5,42 @@
...
-import android.widget.Button;
 import android.widget.GridLayout;
 import android.widget.ImageView;
+import android.widget.Toast;
 
 import java.util.Calendar;
 import java.util.Date;
 
 public class MainActivity extends AppCompatActivity {
+  boolean connectAccount = false; // check if user connect to Account.
```
Comment (nuinuri) [inline Roys/app/src/main/java/com/dwg_karrier/roys/MainActivity.java]: unnecessary. in java, member variable has default value (false)

Comment (nuinuri) [inline Roys/app/src/main/java/com/dwg_karrier/roys/MainActivity.java]: rename into isAccountConnected?

Comment (Csoyee) [inline Roys/app/src/main/java/com/dwg_karrier/roys/MainActivity.java]: I applied comment.

File: Roys/app/src/main/java/com/dwg_karrier/roys/Authentication.java [hunk 5]
```diff
@@ -65,14 +66,13 @@ static String convertStreamToString(InputStream is) {
   }
 
   void authenticationAndBringPages() {
-    Dialog authDialog = new Dialog(mainContext);
+    authDialog = new Dialog(mainContext);
```
Comment (xhae) [inline Roys/app/src/main/java/com/dwg_karrier/roys/Authentication.java]: If authDialog is only declared and used on this method, why does its scope need to place as a member variable?

Comment (Csoyee) [inline Roys/app/src/main/java/com/dwg_karrier/roys/Authentication.java]: As it occurs error. I explained at above commit.

Comment (xhae) [inline Roys/app/src/main/java/com/dwg_karrier/roys/Authentication.java]: In this case, `authDialog` is not accessed outside `authenticationAndBringPages`. With the current situation, I think it's enough to have `final` at here.

Comment (Csoyee) [inline Roys/app/src/main/java/com/dwg_karrier/roys/Authentication.java]: I applied comment

**Comment by xhae** [comment]
Awesome XD

**Pull request merged**